src/core_modules/capture-core/components/DataEntries/common/trackerEvent/withAskToCompleteEnrollment/CompleteModal/completeModal.types.ts

```diff
@@ -0,0 +1,25 @@
+export type ApiEnrollment = any;
+export type ApiEnrollmentEvent = any;
```

Comment:
Never add type local definitions for `ApiEnrollment` and `ApiEnrollmentEvent`, instead use `import type { ApiEnrollment, ApiEnrollmentEvent } from '../../../../../../../capture-core-utils/types/api-types';`